gstreamer-plugin/gstsvthevcenc.c

```diff
@@ -159,7 +158,7 @@ GST_STATIC_PAD_TEMPLATE ("src",
         "width = (int) [ 64, 8192 ], " "height = (int) [ 64, 4320 ], "
         "stream-format = (string) byte-stream, "
         "alignment = (string) au, "
-        "profile = (string) { main, main-10, main-4:4:4}")
+        "profile = (string) { main, main-10, main-4:4:4, main-4:4:4-10 }")
```

Comment:
Hi, @Yeongjin-Jeong, I also added main-444-10 in the profile. and the logic to negotiate.